newIDE/app/src/AssetStore/AssetsList.js

```diff
@@ -808,6 +844,45 @@ const AssetsList = React.forwardRef<Props, AssetsListInterface>(
                     </Link>
                   </Text>
                 )}
+                {privateAssetPackLicense && onPrivateAssetPackSelection && (
+                  <Column noMargin>
+                    <Line noMargin>
+                      <Text size="sub-title">
+                        <Trans>Licensing</Trans>
+                      </Text>
+                      <HelpIcon
+                        size="small"
+                        helpPagePath="https://gdevelop.io/page/asset-store-license-agreement"
+                      />
+                    </Line>
+                    <OwnedProductLicense
+                      productType="asset-pack"
+                      ownedLicense={privateAssetPackLicense}
+                    />
+                    <div style={styles.openProductContainer}>
+                      <FlatButton
+                        label={<Trans>Open product page</Trans>}
```

Comment:
Copy to change I guess, `Product` should not be shown to the users, it must be more specific